### ISSUE TITLE
Focus loss after converting page list to blocks via child editing

### ISSUE THREAD

**Issue by draganescu** [WordPress/gutenberg#49032]
In #48723 the page list improved its UX and it unlocks the tree of pages in the block list view - if the page list is a child of a navigation block. In this case a user can interact with page list item blocks and drag or select them. This interaction results in being prompted to convert the page list block to a list of unsynced navigation link blocks - preserving the tree structure.

This conversion results in focus loss.

This is surprising because if one inserts a page list on its own and uses the toolbar edit button to achieve the same result, there is no focus loss. So, the same code called in different places causes different results.
